mono/mini/mini-exceptions.c

```diff
@@ -2108,6 +2128,15 @@ mono_handle_exception_internal (MonoContext *ctx, MonoObject *obj, gboolean resu
 			free_stack = 0xffffff;
 		}
 
+		if (method->wrapper_type == MONO_WRAPPER_NATIVE_TO_MANAGED && ftnptr_eh_callback) {
```

Comment:
`MONO_FIRST_PASS_CALLBACK_TO_NATIVE` doesn't seem necessary since returning TRUE from handle_exception_first_pass and checking for the wrapper_type again has the same effect.

Reply:
I think the difference might be in how/when we signal the debugger...  we don't want `MONO_FIRST_PASS_CALLBACK_TO_NATIVE` to tell the debugger that we're handling the exception since the callback is kind of internal to the embedder.

Reply:
But obviously I haven't written  a test for that part, so I'm just guessing.

Reply:
Yep, I based this comment on https://github.com/mono/mono/pull/5837/files#r176922871 which was wrong, so please ignore this review.